ibis/tests/sql/test_sqlalchemy.py

```diff
@@ -581,37 +567,14 @@ def test_subquery_aliased(con, star1, star2, snapshot):
     snapshot.assert_match(to_sql(expr), "out.sql")
 
 
-def test_lower_projection_sort_key(con, star1, star2, snapshot):
+def test_lower_projection_sort_key(star1, star2, snapshot):
     t1 = star1
     t2 = star2
 
     agged = t1.aggregate([t1.f.sum().name('total')], by=['foo_id'])
     expr = agged.inner_join(t2, [agged.foo_id == t2.foo_id])[agged, t2.value1]
-    #
```

Comment:
+1, will be easier to maintain

Reply:
I will do some follow ups to move this whole file to snapshot testing.